dts/msm8916/msm8916-unknow-ufi003mb.dts

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+#include <skeleton.dtsi>
+#include <lk2nd.h>
+
+/ {
+ 	model= "Unbranded 4G UFI stick (UFI003_MB_V02)";
+	compatible = "unknow,ufi003-mb", "qcom,msm8916", "lk2nd,device";
+ 	qcom,board-id = <0x08 0x100>;
```

Comment:
This is "msm8916-512mb-mtp" and it looks like there is an "ufi-001b/ufi-001c" board already that seems exactly compatible, given you declare the same button.

Are you sure you *have to* add a new board and can't use lk2nd as-is?

Reply:
> This is "msm8916-512mb-mtp" and it looks like there is an "ufi-001b/ufi-001c" board already that seems exactly compatible, given you declare the same button.
> 
> Are you sure you _have to_ add a new board and can't use lk2nd as-is?
> 
> (...) Thinking of this more, it might still be worth defining this in the same file for lk1st, if the board needs a different DT in Linux than that one.

No, we don't need to. As I said below. The 4 models are very similar to each other. The board design is almost identical. They can share one dts. I got two of them and have seen many photos of the other two. Actually, the only thing to notice is the modem firmware, which can be resolved by msm-firmware-loader or droid-juicer, that has nothing to do with linux kernel or lk2nd/lk1st.